web/src/components/KebabMenu.tsx

```diff
@@ -95,9 +90,9 @@ const KebabMenu = ({ items, direction, show, setShow, Footer }: KebabMenuProps):
 
   return (
     <ToggleContainer>
-      <ToggleButton onClick={onClick} aria-label={t('sideBarOpenAriaLabel')}>
+      <Button onClick={onClick} ariaLabel={t('sideBarOpenAriaLabel')}>
```

Comment:
Would also make sense to use aria-expanded here:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

Reply:
Same as above.